ffi/posix_h.lua

```diff
@@ -53,6 +53,10 @@ ssize_t mq_receive(int, char *, size_t, unsigned int *);
 int mq_close(int) __attribute__((nothrow, leaf));
 int close(int);
 int fcntl(int, int, ...);
+int grantpt(int fd);
+int unlockpt(int fd);
+char *ptsname(int fd);
+int setsid(void);
```

Comment:
That's a very different order. ;-)

Reply:
What are the/your concerns about the order? I have just added what I needed.
We can sort them anyway. What  is a better proposal?

Reply:
I don't know what the order is in the original. I would normally propose to follow that unless it seems grossly illogical for our purposes.

But my point here was that the addition in posix_decl.c should be on line 78 to end up with this result, or conversely that this should be quite a bit further down.

However, as you mentioned this file feels like you gave it more thought while posix_decl.c feels like it's randomly at the bottom, so logic suggests to adjust posix_decl.c. In any case, they should be the same.

Reply:
And if the lines don't match up properly then you can't really spot potential mistakes from such an automated result in the diff because nothing matches. :-)

Reply:
Some historical references: https://github.com/koreader/koreader/pull/4631